### PR TITLE
add `directoryd` and `directory-cli` binary apps

### PR DESCRIPTION
**Pull request by keraliss**
This PR aims to add `directoryd` and `directory-cli` binary apps, and resolve the help text bug that was occurring on running directory server app

### CODE REVIEW

File: src/market/rpc/server.rs [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+) {
+    let rpc_port = directory.rpc_port;
+    let rpc_socket = format!("127.0.0.1:{}", rpc_port);
+    let listener = TcpListener::bind(&rpc_socket).await.unwrap();
+    log::info!(
+        "[{}] RPC socket binding successful at {}",
+        directory.rpc_port,
+        rpc_socket
+    );
+    // read this
```
Comment (mojoX911) [inline src/market/rpc/server.rs:96]: remove this comment. Or change it to something relevant like "start the server loop".

File: src/market/directory.rs [hunk 2]
```diff
@@ -187,6 +190,11 @@ pub async fn start_directory_server(directory: Arc<DirectoryServer>) {
         }
     }
 
+    // start directory rpc server here
```
Comment (mojoX911) [inline src/market/directory.rs:193]: Remove comment.

**Review by mojoX911 — CHANGES_REQUESTED** [review]
Ack. Few nits.

File: src/market/directory.rs (hunk 2 above)
Comment (keraliss) [reply src/market/directory.rs:193]: yeah sorry, completely missed these!

**Review by mojoX911 — APPROVED** [review]
ACK

**Pull request merged**